Dockerfile.windows

```diff
@@ -1,19 +1,11 @@
-FROM dockercore/golang-cross:latest as cli_build
-
-ARG DOCKER_VERSION
-ARG DOCKER_CLI_COMMIT
-RUN mkdir -p /go/src/github.com/docker/cli
-WORKDIR /go/src/github.com/docker/cli
-RUN git clone https://github.com/docker/cli.git . && git checkout $DOCKER_CLI_COMMIT
-RUN VERSION=$DOCKER_VERSION ./scripts/build/windows
-
-FROM debian:jessie
+FROM dockercore/golang-cross:latest
```

Comment:
We'll probably need something like my second commit https://github.com/docker/toolbox/pull/716/commits/fbf5345e3dcffd4ed15f1033e1517696a7ed9f06 to make sure we build using the same Golang version as upstream; wdyt?